src/inference/src/ie_core.cpp

```diff
@@ -96,7 +96,7 @@ Parsed<T> parseDeviceNameIntoConfig(const std::string& deviceName, const std::ma
     } else if (deviceName_.find("MULTI:") == 0) {
         deviceName_ = "MULTI";
         config_[ie::MultiDeviceConfigParams::KEY_MULTI_DEVICE_PRIORITIES] = deviceName.substr(6);
-    } else if (deviceName.find("AUTO") == 0) {
+    } else if (deviceName == "AUTO" || deviceName.find("AUTO:") == 0) {
```

Comment:
can you explain more what's the logic been handled behind this?

Reply:
avoid parsing key AUTO_BATCH_TIMEOUT to device AUTO